packages/react-core/src/components/Toolbar/examples/ToolbarGroups.tsx

```diff
@@ -0,0 +1,158 @@
+import React from 'react';
+import { Toolbar, ToolbarContent, ToolbarGroup, ToolbarItem } from '@patternfly/react-core';
+import { Button, Select, SelectOption, SelectVariant } from '@patternfly/react-core';
+import EditIcon from '@patternfly/react-icons/dist/esm/icons/edit-icon';
+import CloneIcon from '@patternfly/react-icons/dist/esm/icons/clone-icon';
+import SyncIcon from '@patternfly/react-icons/dist/esm/icons/sync-icon';
+
+export const ToolbarGroups: React.FunctionComponent = () => {
+  const firstOptions = [
+    { value: 'Filter 1', disabled: false, isPlaceholder: true },
+    { value: 'A', disabled: false },
+    { value: 'B', disabled: false },
+    { value: 'C', disabled: false }
+  ];
+
+  const secondOptions = [
+    { value: 'Filter 2', disabled: false, isPlaceholder: true },
+    { value: '1', disabled: false },
+    { value: '2', disabled: false },
+    { value: '3', disabled: false }
+  ];
+
+  const thirdOptions = [
+    { value: 'Filter 3', disabled: false, isPlaceholder: true },
+    { value: 'I', disabled: false },
+    { value: 'II', disabled: false },
+    { value: 'III', disabled: false }
+  ];
+
+  const [firstIsExpanded, setFirstIsExpanded] = React.useState(false);
+  const [firstSelected, setFirstSelected] = React.useState();
+  const [secondIsExpanded, setSecondIsExpanded] = React.useState(false);
+  const [secondSelected, setSecondSelected] = React.useState();
+  const [thirdIsExpanded, setThirdIsExpanded] = React.useState(false);
+  const [thirdSelected, setThirdSelected] = React.useState();
+
+  const onFirstToggle = (isExpanded: boolean) => {
+    setFirstIsExpanded(isExpanded);
+  };
+
+  const onFirstSelect = (event, selection) => {
```

Comment:
Missing types for these arguments, also I would add the conventional `_` to the front of event since it isn't being used. I would have the same note for a lot of other functions across several of these examples.